ravenframework/Models/PostProcessors/SparseSensing.py

```diff
@@ -48,14 +49,26 @@ class cls.
                                                   descr=r"""The goal of the sparse sensor optimization (i.e., reconstruction or classification)""")
     goal.addParam("subType", InputTypes.makeEnumType("Goal", "GoalType", ['reconstruction','classification']), False, default='reconstruction')
     inputSpecification.addSub(goal)
-    features = InputData.parameterInputFactory("features", contentType=InputTypes.StringListType,
+    features = InputData.parameterInputFactory("features", contentType=InputTypes.StringListType, ##Change with variable X,Y
                                                 printPriority=108,
                                                 descr=r"""Features/inputs of the data model""")
     goal.addSub(features)
-    target = InputData.parameterInputFactory("target", contentType=InputTypes.StringType,
+    xValue = InputData.parameterInputFactory("xValue", contentType=InputTypes.StringListType,
                                                 printPriority=108,
-                                                descr=r"""target of data model""")
-    goal.addSub(target)
+                                                descr=r"""Variable plotted on the X-axis of the data model""")
+    goal.addSub(xValue)
+    yValue = InputData.parameterInputFactory("yValue", contentType=InputTypes.StringListType,
+                                                printPriority=108,
+                                                descr=r"""Variable plotted on the Y-axis of the data model""")  ##Do we want another Z-value
```

Comment:
Hm, do we want a zValue?

Reply:
Yes! In case we have 3D data! Thank you for catching that!